listenbrainz_spark/recommendations/utils.py

```diff
@@ -2,11 +2,18 @@
 
 from jinja2 import Environment, FileSystemLoader
 
+HTML_FILES_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'html_files')
+
 def save_html(filename, context, template):
     path = os.path.dirname(os.path.abspath(__file__))
     template_environment = Environment(
     loader=FileSystemLoader(os.path.join(path, 'templates')))
-    outputfile = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'html_files', filename)
+    outputfile = os.path.join(HTML_FILES_PATH, filename)
     with open(outputfile, 'w') as f:
         html = template_environment.get_template(template).render(context)
         f.write(html)
+
+
+def check_html_files_dir_path():
```

Comment:
You could just do this test in the `save_html` function above, and then the users of the save html function don't have to worry about it.

Reply:
Ah! I don't know what I was thinking whilst writing it 🙄. Thank you 🌹